### PR TITLE
A0-1530 Rename kickouts to Suspensions

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
* Rename Kick-outs tabs to Suspension tab
* change parsing of BanValidators events to use information about `start` in `era` column
* simplify logic of `useSuspensions` hook

Example of how suspension looks like:

image

### CODE REVIEW

**Review by kostekIV — APPROVED** [review]
lgtm, one comment for the future it would make it easier to review if this would be split into two prs
1. renaming
2. changing useSuspensions hook

File: packages/page-staking/src/Suspensions/Suspensions.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+import { createNamedHook, useApi } from '@polkadot/react-hooks';
+import { Vec } from '@polkadot/types';
+import { EventRecord, Hash } from '@polkadot/types/interfaces';
+import { Codec } from '@polkadot/types/types';
+
+type SuspensionReasons = [string, string, number][];
+
+function parseEvents (events: EventRecord[]): SuspensionReasons {
+  return events.filter(({ event }) => {
+    return event.section === 'elections' && event.method === 'BanValidators';
```
Comment (deuszx) [inline packages/page-staking/src/Suspensions/Suspensions.tsx]: isn't this equivalent to
 ```javascript
event.section === 'elections' && event.method === 'BanValidators'
```

File: packages/page-staking/src/Suspensions/Suspensions.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+      const reasonValue = Object.values(reasonTypeAndValue)[0];
+      const era = Number(reasonAndEra.start.toString());
+
+      if (reasonType === 'OtherReason') {
+        return [account, reasonValue, era];
+      } else if (reasonType === 'InsufficientUptime') {
+        return [account, 'Insufficient uptime in at least ' + reasonValue + ' sessions', era];
+      } else {
+        return [account, reasonType + ': ' + reasonValue, era];
+      }
```
Comment (deuszx) [inline packages/page-staking/src/Suspensions/Suspensions.tsx]: Since you return from every clause you shouldn't need the 	`return` keyword?

**Review by DamianStraszak — APPROVED** [review]
Looks great. I was thinking that the last piece of info that is missing is the "suspension period" (i.e. 10 eras for us). Is it possible to display it somewhere? And we could make is very simple, like just display the current setting and don't care about changes in the past.

File: packages/page-staking/src/Suspensions/Suspensions.tsx (hunk 1 above)
Comment (Marcin-Radecki) [reply packages/page-staking/src/Suspensions/Suspensions.tsx]: Yes, will fix

File: packages/page-staking/src/Suspensions/Suspensions.tsx (hunk 2 above)
Comment (Marcin-Radecki) [reply packages/page-staking/src/Suspensions/Suspensions.tsx]: I think as long as I use `{}` in `map` , I need to specify explicitly `return`

**Pull request merged**